services/MerchStoreService.ts

```diff
@@ -884,6 +884,14 @@ export default class MerchStoreService {
     return moment().isSame(moment(pickupEvent.start), 'day');
   }
 
+  private static isFuturePickupEvent(pickupEvent: OrderPickupEventModel): boolean {
+    return moment().isBefore(moment(pickupEvent.start), 'day');
+  }
+
+  private static isPastPickupEvent(pickupEvent: OrderPickupEventModel): boolean {
+    return moment().isAfter(moment(pickupEvent.start), 'day');
+  }
+
```

Comment:
If we specify the granularity `'day'`, then we are only comparing the days of the two rather than the actual times. So if the pickup event is at 4pm and it's 2pm right now, then `isFuturePickupEvent` would be false because today is not before today. So we don't want to specify the granularity of `day`. Can we add a test to check that we can't complete a pickup event on the same day if the time is before the pickup event?

also nit: I don't think isPastPickupEvent is being used so let's remove it